wdl/SelectBatchesAndColumns.wdl

```diff
@@ -0,0 +1,46 @@
+version 1.0
+
+task SelectBatchesAndColumns {
+	input {
+		File sample_sets_table_tsv
+		Array[String]? batches_opt
+		String column
+		String? docker
+	}
+	
+	Boolean select_batches=defined(batches_opt)
+	Array[String] batches=select_first([batches_opt, ["all"]])
+
+	String docker_set = select_first([docker, "ubuntu:18.04"])
+
+	runtime {
+		docker: docker_set
```

Comment:
I believe the default is 3.75 GiB RAM which is probably more than you need. I would suggest reducing to 0.9 GiB.

Reply:
Also 3 `preemptible_tries` and 1 `max_retries`